R/pmcmc.R

```diff
@@ -696,3 +699,165 @@ plot.pmcmc <- function(x, ...) {
   
   
 }
+
+##' @export
+##' @importFrom viridis cividis
+##' @importFrom graphics hist par plot.new text lines legend
+##' 
+plot.pmcmc_list <- function(x, burn_in = 1, ...) {
+  
+  summ <- summary(x, burn_in = burn_in)
+  par_names <- names(x$inputs$pars$pars_init)
+  n_pars <- length(par_names)
+  
+  chains <- x$chains
+  n_chains <- length(chains)
+  cols_trace <- viridis::viridis(n_chains)[c(4, 1, 3, 2)]
+  
+  
+  # compile master chain and order by log posterior for plotting
+  master_chain <- do.call(what = rbind, 
```

Comment:
We use this function a couple of times (possibly in ncov outputs too). I think it would be useful to make into a method for `mcmc_list` and export it

Reply:
I agree, it's been bugging me for a little while, thanks for the prompt